jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -57,8 +53,8 @@ pipeline {
         stage('detect docker image + args') {
             agent {
                 docker {
-                    label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
-                    image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028'
+                    label AGENT_LABEL
+                    image 'alpine:latest'
```

Comment:
Hey can we have the alpine tag with fixed version?, as latest every-time can have chances to break, we can go with [3.15.4](https://hub.docker.com/layers/alpine/library/alpine/3.15.4/images/sha256-a777c9c66ba177ccfea23f2a216ff6721e78a662cd17019488c417135299cd89?context=explore)

Reply:
Other than this PR looks good for me.

Reply:
What about `alpine:3`

Reply:
Fixed now.